.github/build-cpp-filecoin.src.yml

```diff
@@ -0,0 +1,204 @@
+name: Fuhon (cpp-Filecoin)
+
+on:
+  push:
+    branches: [master, feature/github-action-tag-and-publish-docker-image]
+    tags: '**'
+  pull_request:
+    branches: [master, feature/github-action-tag-and-publish-docker-image]  ## target branches
```

Comment:
Will `feature/github-action-tag-and-publish-docker-image` be removed from these lists before merging into `master`?

Or will new commits be pushed to `feature/github-action-tag-and-publish-docker-image` after PR is merged?

Reply:
the branch will be removed, it is there now just for testing. (will create a new PR specifically for it).